app/views/public_reviews/new_reviewer.html.haml

```diff
@@ -12,4 +12,5 @@
   = f.hidden_field :roles, value: "recenzent", label: "Role", name: "person[roles][]"
  
   = f.submit 'Dodaj propozycję', class: %w{btn btn-primary btn-outline}
-= button_to 'Zakończ dodawanie recenzentów', finish_public_reviews_path, method: :get
+  = button_to 'Zakończ dodawanie recenzentów', finish_public_reviews_path, method: :get
```

Comment:
Z tego co pamiętam, to ten guzik miał być właśnie w tym miejscu - być może stąd bierze się błąd.